### ISSUE TITLE
Counting connections

### ISSUE THREAD

**Issue by everanium** [jhuckaby/performa-satellite#1]
Hello.

Some my servers have over the 10000 established connections.

In this case performa-satellite send 0 connections to Performa Server, and I have zero Open Tcp Connections graph. Screenshots in attachment.

I apologize this is may be problem in npm module:

https://github.com/sebhildebrandt/systeminformation/blob/master/lib/network.js

`exec(cmd, { maxBuffer: 1024 * 2000 }, function (error, stdout)`

![zero](https://user-images.githubusercontent.com/38362418/59915760-9aab0980-9447-11e9-958b-4045a50b34e3.png)
![zero-more](https://user-images.githubusercontent.com/38362418/59915932-08efcc00-9448-11e9-9e34-efe81bb10b31.png)

**Comment by jhuckaby** [comment]
Thanks, I opened an issue with `systeminformation`: https://github.com/sebhildebrandt/systeminformation/issues/250

**Comment by jhuckaby** [comment]
Fixed in Performa Satellite [v1.0.7](https://github.com/jhuckaby/performa-satellite/releases/tag/v1.0.7).  You'll have to reinstall this on all your servers via:

```
curl -L https://github.com/jhuckaby/performa-satellite/releases/download/v1.0.7/performa-satellite-linux > /opt/performa/satellite.bin
chmod 755 /opt/performa/satellite.bin
```
